Resources/Prototypes/Recipes/Construction/Graphs/glass.yml

```diff
@@ -0,0 +1,50 @@
+- type: constructionGraph
+  id: glass
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: SheetGlass
+        - to: SheetRGlass
+        - to: SheetPGlass
+        - to: SheetRPGlass
```

Comment:
Why do these edges not have steps? This will allow people to craft them for free, or just not craft them at all...